test/serializer/links_test.rb

```diff
@@ -11,7 +11,11 @@ def setup
       end
 
       def test_item_links_attributes
-        expected = [dashboard: { href: "http://example.com/my-dashboard" }]
+        expected = [
+          { dashboard: { href: "http://example.com/my-dashboard" } },
+          { avatar: { :href=>"http://example.com/avatar.jpg" } }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.